packages/osd-ui-shared-deps/flot_charts/API.md

```diff
@@ -575,7 +575,7 @@ In Python you can get it with something like:
 calendar.timegm(datetime_object.timetuple()) * 1000
 ```
 In Ruby you can get it using the `#to_i` method on the
-[`Time`](http://apidock.com/ruby/Time/to_i) object. If you're using the
+`Time` object. If you're using the
```

Comment:
are we planning to add new link or existing link in the future?

Reply:
No - it's just because we internalized this library (including the docs) - we will never have any need to generate compatible timestamps in ruby 😁